openapi.yml

```diff
@@ -152,6 +152,9 @@ components:
         imageUrl:
           type: string
           description: The primary image for a Recommendation.
+        timeToRead:
+          type: integer
+          description: Article read time in minutes
```

Comment:
@ScottDowne Just to confirm, `timeToRead` (camel-case) is supported by Firefox when hitting the new api, right?

Seems like that's the case looking at the code: https://searchfox.org/mozilla-central/source/browser/components/pocket/content/pktApi.sys.mjs#868

Reply:
That is correct.